interfaces/xenmgr_vm.xml

```diff
@@ -71,10 +72,10 @@
       </tp:docstring>
     </property>
     <property name="initrd" type="s" access="readwrite"/>
-    <property name="acpi-pt" type="b" access="readwrite">
+    <property name="acpi-path" type="s" access="readwrite">
```

Comment:
Now that this is a string, what does it contain? It used to just turn on the acpi-pt bits in xenvm which happened to only pass the SLIC (thus originally being a bad name).